var/spack/repos/builtin/packages/glm/package.py

```diff
@@ -14,25 +14,33 @@ class Glm(CMakePackage):
     homepage = "https://github.com/g-truc/glm"
     url = "https://github.com/g-truc/glm/archive/0.9.9.8.tar.gz"
 
+    version('develop', git="https://github.com/g-truc/glm.git")
```

Comment:
Can we move the git property at the class level and use `branch=...` in the version directive?

Reply:
Thanks for the review.  Fixed.